File: RepoTransBench/target_projects/Java/C#/l42111996_java-Kcp/public_tests/KcpServerExamplesPublicTest.cs
using Xunit;

namespace KcpPublicTests
{
    public class KcpServerExamplesPublicTest
    {
        [Fact]
        public void TestSimplePublicServer()
        {
            int serverId = 42;
            int port = 16667;
            string expectedGreeting = "KCP Public Test Server Started (ServerID: 42, Port: 16667)";

            string serverGreeting = SimulateServerStart(serverId, port);
            Assert.Equal(expectedGreeting, serverGreeting);
        }

        private string SimulateServerStart(int serverId, int port)
        {
            return $"KCP Public Test Server Started (ServerID: {serverId}, Port: {port})";
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/l42111996_java-Kcp/public_tests/kcp_server_examples_public_test.go
package public_tests

import (
	"testing"
)

func TestSimplePublicServer(t *testing.T) {
	serverID := 42
	port := 16667
	expectedGreeting := "KCP Public Test Server Started (ServerID: 42, Port: 16667)"
	greeting := simulateServerStart(serverID, port)
	if greeting != expectedGreeting {
		t.Errorf("Expected '%s', got '%s'", expectedGreeting, greeting)
	}
}

func simulateServerStart(serverId int, port int) string {
	return "KCP Public Test Server Started (ServerID: " + itoa(serverId) + ", Port: " + itoa(port) + ")"
}

func itoa(i int) string {
	// Fast path for small ints
	var digits = "0123456789"
	if i == 0 {
		return "0"
	}
	neg := i < 0
	if neg {
		i = -i
	}
	result := [20]byte{}
	pos := len(result)
	for i > 0 {
		pos--
		result[pos] = digits[i%10]
		i /= 10
	}
	if neg {
		pos--
		result[pos] = '-'
	}
	return string(result[pos:])
}